### PR TITLE
Dev 565 httpsprebuilt uinetlifyapp polish

### PR DESCRIPTION
**Pull request by kimberleejohnson**
- Style tweaks to: `.container`, `.wrapper`, `.white-button`, `.start-container` 
- Toggle participant bar clarity 
- Added title to showLocalVideo too 
- First pass at error states: maybe not most efficient but they seem to work! 

To see error states: 
- Visit the preview link and enter some fake text to get the "Invalid URL" message: https://deploy-preview-7--prebuilt-ui.netlify.app/ 
- Run this branch locally and click "Create room" without changing any of the information in index.js

### CODE REVIEW

File: index.js
Comment (jessmitch42) [inline index.js:117]: Does it go into this block for you? I'm seeing this when I pass a random string as the Daily Url when I use the preview link. 
CleanShot 2021-05-19 at 18 30 51@2x

File: index.html
Comment (jessmitch42) [inline index.html:88]: I wonder if instead of reloading you want to just clear the error state and go back to the form? If the URL is invalid it may have just been a typo, so no reload necessary. (In this case, just "Try again" should work for the button.) I'll leave to you to decide, though!

File: index.js
Comment (jessmitch42) [inline index.js:88]: I think double-check is hyphenated. Just "check" would work too 🤷‍♂️ Whatever you think

File: index.css
Comment (jessmitch42) [inline index.css:330]: Mind adding this too? The benefit is for long strings or small screens, it will stay centered if the text wraps to a second line and won't touch the window edges.
```
padding: 0 16px;
text-align: center;
```

I'm also a big fan of max-widths on text like this to force wrap it if it's long but that's not super necessary if we're controlling the length of the message. Just mentioning it for a btw.

**Review by jessmitch42 — CHANGES_REQUESTED** [review]
Nice job! I'm not seeing the error for invalid meeting URLs so I'll hold off on approving until you confirm if it's working on your end.

File: index.js
Comment (kimberleejohnson) [reply index.js:88]: Thank you! "check" it is.

File: index.js
Comment (kimberleejohnson) [reply index.js:117]: It _did_! And this is why I should screen record everything, because now I feel like I'm losing my mind. I'll debug!

File: index.css
Comment (kimberleejohnson) [reply index.css:330]: Added, thank you!

File: index.html
Comment (kimberleejohnson) [reply index.html:88]: This is a great idea. I added, thank you!

File: index.js
Comment (kimberleejohnson) [reply index.js:117]: https://www.loom.com/share/397a7f7e54f14d2d8d153c5903ef5df7

Comment (kimberleejohnson) [reply index.js:116]: https://www.loom.com/share/e9e2ccb53341403c982371689afb3458

Comment (jessmitch42) [reply index.js:116]: Thanks for the looms, super helpful. This can be another ticket (non-blocking) but it would be great to add a `disabled` attribute to the button instead of just styling it disabled. I actually didn't know it was disabled until seeing you describe it here. I thought it was just styled like a secondary button and didn't do anything. 😂 So that was part of the confusion. 

LGTM-- approving! Nice job.

**Review by jessmitch42 — APPROVED** [review]
🎉

File: index.js
Comment (kimberleejohnson) [reply index.js:116]: Thank you! I'm working on adding that disabled attribute (seems very quick), and will merge after.

**Pull request merged**
